app/views/external_users/claims/case_details/_transfer_court_fields.html.haml

```diff
@@ -8,4 +8,7 @@
   = f.collection_select :transfer_court_id, Court.alphabetical, :id, :name, { include_blank: ''.html_safe }, { class: 'form-control fx-autocomplete', 'aria-label': t('.transfer_court') }
 
 .form-group.js-typeahead{ class: error_class?(@error_presenter, :transfer_case_number) }
-  = f.adp_text_field :transfer_case_number, label: t('.transfer_case_number'), hint_text: t('.transfer_case_number_hint'), input_classes: '',  errors: @error_presenter
+  - if Settings.urn_enabled?
```

Comment:
[see above](https://github.com/ministryofjustice/Claim-for-Crown-Court-Defence/pull/3457/files#r472029346)

Reply:
👍 thank you for the suggestion, I have amended to use dynamic translations (option 2).